dist/css/accordion.css

```diff
@@ -7,6 +7,7 @@
   background-image: url("../assets/plus_round_red.png");
   background-size: 28px;
   background-position: 97% 50%;
+  -webkit-box-shadow: none;
```

Comment:
what level of support are you going for here?
https://caniuse.com/#search=box-shadow

Reply:
prefixer add-in